frontend/src/store/modules/auth/mutations.js

```diff
@@ -1,18 +1,20 @@
-import StorageService from "../../../services/common/storageService";
+import StorageService from '../../../services/common/storageService';
 
 export default {
     SET_TOKEN: (state, token) => {
         state.token = token;
     },
     USER_LOGIN: (state, response) => {
-        StorageService.setToken(response.token);
-        state.token = response.token;
+        StorageService.setToken(response.access_token);
+        state.token = response.access_token;
         state.currentUser = response.user;
```

Comment:
'response' doesn't have 'user'

Reply:
Yes, Stanislav rights, you need to call fetchAuthenticatedUser action after success login, like this:

```
 login: (context, user) => {
        httpService.post('auth/login', {
            email: user.email,
            password: user.password
        })
            .then((res) => {
                context.commit('USER_LOGIN', res.data.data);
                context.dispatch('fetchAuthenticatedUser', res.data.data.access_token);
            }).catch(function (err) {
                // TODO: Handle error
            });
    },
```